addons/event_sale/migrations/9.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,27 @@
+---Fields in module 'event_sale'---
+event_sale   / event.event              / badge_back (html)             : module is now 'event' ('event_sale')
+# Nothing to do
+
+event_sale   / event.event              / badge_innerleft (html)        : module is now 'event' ('event_sale')
+# Nothing to do
+
+event_sale   / event.event              / badge_innerright (html)       : module is now 'event' ('event_sale')
+# Nothing to do
+
+event_sale   / event.registration       / sale_order_id (many2one)      : NEW relation: sale.order
```

Comment:
@hbrunn 
Maybe it works with https://github.com/OCA/OpenUpgrade/pull/713/commits/4ece55f80371e1ecfacde123eae84e340c5227a0